pkg/provider/kubernetes/kubernetes.go

```diff
@@ -231,5 +240,25 @@ func (k *BaseClient) fetchSecretKey(ctx context.Context, key esmeta.SecretKeySel
 }
 
 func (k *ProviderKubernetes) Validate() error {
+	ctx := context.Background()
+
+	authReview, err := k.ReviewClient.Create(ctx, &authv1.SelfSubjectAccessReview{
```

Comment:
Can we add a small note to the docs that the `SelfSubjectAccessReview` permission is required for the service account?

Reply:
@moolen thank you for your feedback. I have added a small note to the documentation.